modules/simple-payments/simple-payments.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+class Jetpack_Simple_Payments {
+	// These have to be under 20 chars because that is CPT limit.
+	static $post_type_order = 'jp_pay_order';
+	static $post_type_product = 'jp_pay_product';
+
+	static $shortcode = 'simple-payment';
+
+	// Classic singleton pattern:
+	private static $instance;
+	private function __construct() {}
+	static function getInstance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+			self::$instance->register_init_hook();
+		}
+		return self::$instance;
+	}
+
+	private function register_scripts() {
+		wp_register_script( 'paypal-checkout-js', 'https://www.paypalobjects.com/api/checkout.js' );
+		wp_register_script( 'paypal-express-checkout', plugins_url( '/paypal-express-checkout.js', __FILE__ ) , array( 'paypal-checkout-js' ) );
```

Comment:
Do we need this script? On https://developer.paypal.com/docs/integration/direct/express-checkout/integration-jsv4/server-side-REST-integration/, I see only the first one. Or is it our own JS script for this shortcode?